src/Features/VisualBasic/Portable/xlf/VBFeaturesResources.zh-Hant.xlf

```diff
@@ -332,21 +332,6 @@
         <target state="translated">選取範圍無效</target>
         <note />
       </trans-unit>
-      <trans-unit id="Selection_doesn_t_contain_any_valid_token">
```

Comment:
can you make the loc change in a separate PR?

Reply:
#55291